components/button.css

```diff
@@ -0,0 +1,92 @@
+.btn {
+  background-color: transparent;
+  font-size: 1.2rem;
+  font-weight: 700;
+  border-radius: 5px;
+  padding: 0.5rem 1rem;
+  border: none;
+  cursor: pointer;
+  margin-bottom: 1rem;
+}
+/* 
```

Comment:
Remove comments  while merging. Rest code looks clean and good

Reply:
Thanks for pointing it out. Removed unnecessary comments